performance_regression_row_level_repair_test.py

```diff
@@ -0,0 +1,483 @@
+
+    # Util functions ===============================================================================================
+
+
+    # Global Util functions ===============================================================================================
+
+    def get_used_capacity(self, node):
+        # (sum(node_filesystem_size{mountpoint="/var/lib/scylla"})-sum(node_filesystem_avail{mountpoint="/var/lib/scylla"}))
+        filesystem_capacity_query = 'sum(node_filesystem_size{{mountpoint="{0.scylla_dir}", ' \
+                                    'instance=~"{1.private_ip_address}"}})'.format(self, node)
+
+        self.log.debug("filesystem_capacity_query: {}".format(filesystem_capacity_query))
+
+        fs_size_res = self.prometheusDB.query(query=filesystem_capacity_query, start=time.time(), end=time.time())
+        kb_size = 2 ** 10
+        mb_size = kb_size * 1024
+        gb_size = mb_size * 1024
+        fs_size_gb = int(fs_size_res[0]["values"][0][1]) / gb_size
+        self.log.debug("fs_cap_res: {}".format(fs_size_res))
+        used_capacity_query = '(sum(node_filesystem_size{{mountpoint="{0.scylla_dir}", ' \
+                              'instance=~"{1.private_ip_address}"}})-sum(node_filesystem_avail{{mountpoint="{0.scylla_dir}", ' \
+                              'instance=~"{1.private_ip_address}"}}))'.format(self, node)
+
+        self.log.debug("used_capacity_query: {}".format(used_capacity_query))
+
+        used_cap_res = self.prometheusDB.query(query=used_capacity_query, start=time.time(), end=time.time())
+        self.log.debug("used_cap_res: {}".format(used_cap_res))
+
+        assert used_cap_res, "No results from Prometheus"
+        used_size_mb = float(used_cap_res[0]["values"][0][1]) / float(mb_size)
+        used_size_gb = float(used_size_mb / 1024)
+        self.log.debug(
+            "The used filesystem capacity on node {} is: {} MB/ {} GB".format(node.public_ip_address, used_size_mb,
+                                                                              used_size_gb))
+        return used_size_mb
+
+    @retrying(n=80, sleep_time=60, allowed_exceptions=(AssertionError,))
+    def _wait_no_compactions_running(self):
+        q = "sum(scylla_compaction_manager_compactions{})"
+        now = time.time()
+        results = self.prometheusDB.query(query=q, start=now - 60, end=now)
+        self.log.debug("scylla_compaction_manager_compactions: %s" % results)
+        # if all are zeros the result will be False, otherwise there are still compactions
+        if results:
+            assert any([float(v[1]) for v in results[0]["values"]]) is False, \
+                "Waiting until all compactions settle down"
+
+    def _disable_hinted_handoff(self):
+
+        yaml_file = "/etc/scylla/scylla.yaml"
+        for node in self.db_cluster.nodes:  # disable hinted handoff
+            res = node.remoter.run('sudo echo hinted_handoff_enabled: false >> {}'.format(yaml_file))
```

Comment:
can we do this the same in with the configuration, i.e. with sed ?, this can end up having `hinted_handoff_enabled` written multiple times in scylla.yaml, and might cause confusion.

Reply:
'sed' is not good in case parameter doesn't exist in yaml. fixed this in another way with 'grep -v'.